portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -72,6 +86,22 @@ private String convertToJsonUsingGson(List messages) {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
+    //TODO: Move into its own function.
+    //Populate the messages array.
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("Task");
+    PreparedQuery results = datastore.prepare(query);
+
+    //Iterate results.
+    for (Entity entity : results.asIterable()) {
+
```

Comment:
Nit: Remove blank line at top of for loop body.

Reply:
Done!